docs/Performance/promise.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: default
+title: Promise
+name: Overview
+parent: Performance
+nav_order: 1
+---
+
+## Promise
+
+If you are new to Promises, I would suggest to make yourself familiar before continuing:
+
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise>
+
```

Comment:
and what do you think about adding some introductory prose never the less?
along the lines of
- promises help controlling exec order of async operations
- promises don't block the browser event loop while execution &rarr; 
  effect on UX as `busy` indicators can be used nicely to indicate elements being rehyrated with data
- ...

Reply:
this is a good idea, added this in one of the latest commits